core/transport/src/main/proto/grpc/block_worker.proto

```diff
@@ -155,6 +155,7 @@ message UfsReadOptions{
   // We introduce a heuristic to choose which API to use.
   required bool position_short = 2;
   optional int64 bandwidth = 3;
+  optional string user = 4;
```

Comment:
@madanadit @yuzhu is it fine to move this field there?

Reply:
user is necessary to control bandwidth and there's no related security problem with this part since we don't have related support in UFS

Reply:
I see,  just want to make sure we are aligned.